lang/src/org/partiql/lang/CompilerPipeline.kt

```diff
@@ -264,6 +267,15 @@ internal class CompilerPipelineImpl(
     override val plugin: Plugin
 ) : CompilerPipeline {
 
+    // Initialize a map from a type alias to a scalar type, as part of work for PartiQL compile pipeline to install the plugin
+    private val aliasesToScalarType: Map<String, ScalarType>
+
+    init {
+        plugin.validate()
+
+        aliasesToScalarType = plugin.mapAliasToScalarType()
+    }
```

Comment:
We should not be assuming there is only one plugin. We should also namespace any identifiers named within a plugin

Reply:
For now we only consider there is only one plugin for approaching pluggability of OTS.